docs/source/tutorials/blink.rst

```diff
@@ -0,0 +1,111 @@
+##########################
+Blink Interactive Tutorial
+##########################
+
+Introduction
+------------
+
+This is the interactive blink tutorial using `Wokwi`_. For this tutorial, you don't need the ESP32 board or the Arduino toolchain.
+
+.. note:: If you don't want to use this tutorial with the simulation, you can copy and paste the :ref:`blink_example_code` from `Wokwi`_ editor and use on the Arduino IDE or PlatformIO.
```

Comment:
Please check if link to "blink_example_code" works? In preview mode doesn't work.

Links to Arduino IDE and PlatformIO Documentation pages can be added.

[Arduino IDE](https://docs.espressif.com/projects/arduino-esp32/en/latest/installing.html#installing-using-boards-manager)
[PlatformIO](https://docs.espressif.com/projects/arduino-esp32/en/latest/installing.html#platformio)